plugins/woocommerce-blocks/tests/e2e/tests/filter-blocks/active-filters.block_theme.spec.ts

```diff
@@ -28,8 +28,6 @@ test.describe( 'Product Filter: Active Filters Block', async () => {
 			page,
 			defaultBlockPost,
 		} ) => {
-			await page.goto( defaultBlockPost.link );
-
```

Comment:
(This comment refers to lines 37-38 of this file. I couldn't leave it there as it's apparently "not a part of the diff", so leaving it here 🤷)

`locator.count()` is inherently unstable as it returns the immediately available number of elements, while the rest might still be loading. Let's use the dedicated [toHaveCount](https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-have-count) assertion instead:

```js
await expect( locator ).toHaveCount( 1 );
```

Reply:
Yeah its not part of the diff because I'm just refactoring the template stuff here, but I don't mind adding in some improvements to the tests too. This is a simple fix, so I'll add that.

Reply:
Fixed in https://github.com/woocommerce/woocommerce/pull/44914/commits/93ab9c408760c1347f98617a8a92ea20c8ae1d69